packages/main/src/Slider.js

```diff
@@ -265,8 +270,12 @@ class Slider extends SliderBase {
 		return this.value.toFixed(stepPrecision);
 	}
 
-	get tabIndexProgress() {
-		return "-1";
+	get _ariaDisabled() {
+		return this.disabled || undefined;
+	}
+
+	get ariaLabelledByText() {
```

Comment:
Add an underscore indicating a private property like the rest of the aria related ones.